adcc/AdcBlockView.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+## vi: tabstop=4 shiftwidth=4 softtabstop=4 expandtab
+## ---------------------------------------------------------------------
+##
+## Copyright (C) 2020 by the adcc authors
+##
+## This file is part of adcc.
+##
+## adcc is free software: you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published
+## by the Free Software Foundation, either version 3 of the License, or
+## (at your option) any later version.
+##
+## adcc is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with adcc. If not, see <http://www.gnu.org/licenses/>.
+##
+## ---------------------------------------------------------------------
+import numpy as np
+
+from .AdcMatrix import AdcMatrixlike
+from .functions import empty_like
+from .AmplitudeVector import AmplitudeVector
+
+import libadcc
+
+
+class AdcBlockView(AdcMatrixlike):
+    def __init__(self, fullmatrix, block):
+        """
+        Object which acts as a single block of the ADC matrix.
```

Comment:
Please provide an example for `block`. Is it a string like `ss`, `sd`, `ds`, and `dd`?

Reply:
No it's just "s" or "d", because that's the easiest to code. I mean for example for "ds" there is no expression in libadc afaik, so implementing that gets kind of annoying. We could leave that for future work, however. I do not see for what we would need that at the moment, however. Do you have a use case in mind.